Malmo/src/PythonWrapper/CMakeLists.txt

```diff
@@ -27,11 +27,15 @@ set( OTHER_FILES
 
 python_add_module( MalmoPython SHARED ${CPP_SOURCES} )
 include_directories( MalmoPython ${PYTHON_INCLUDE_DIRS} )
-target_link_libraries( MalmoPython Malmo ${PYTHON_LIBRARIES} )
+target_link_libraries( MalmoPython Malmo )
 if( MSVC )
   set_property( TARGET MalmoPython APPEND PROPERTY COMPILE_DEFINITIONS "BOOST_PYTHON_STATIC_LIB=1" ) 
+  target_link_libraries( MalmoPython Malmo ${PYTHON_LIBRARIES} )
 elseif( APPLE )
-  set_property( TARGET MalmoPython PROPERTY SUFFIX ".so" )
+  set_target_properties(MalmoPython
+      PROPERTIES
+      SUFFIX ".so"
+      LINK_FLAGS "-undefined dynamic_lookup")
```

Comment:
This is needed for MacOS but has the downside of the linker not reporting undefined symbols until runtime. Do we 100% need it for Linux builds as it would be so much better to get errors during build?

Reply:
We don't need this flag for Linux that's why it's under the if block for APPLE

Reply:
Ah good. Sorry.